scripts/extensions/pool_update.apply

```diff
@@ -53,6 +53,7 @@ def execute_apply(session, update_package, yum_conf_file):
     cmd = ['yum', 'clean', 'all', '--noplugins', '-c', yum_conf_file]
     p = subprocess.Popen(cmd, shell=False, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, close_fds=True, env=yum_env)
     output, _ = p.communicate()
+    output = output.decode('utf-8')
```

Comment:
would it be better to call `Popen` with `encoding='utf-8'` so stdout is opened in text mode?

Reply:
Yes, you're right. Corrected!

Reply:
`universal_newlines` is often used to get the `string` output.

Reply:
Corrected and tested!

Reply:
probably better to use `text=True` as opposed to `universal_newlines` from [here](https://docs.python.org/3/library/subprocess.html#popen-constructor)

> New in version 3.7: text was added as a more readable alias for universal_newlines.